<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Consommateurig</class>
 <widget class="QWidget" name="Consommateurig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1303</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="1">
    <widget class="QListWidget" name="Livraison"/>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="Confirmer">
     <property name="text">
      <string>Confirmer</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="LivraisonLabel">
     <property name="text">
      <string>LIvraison   Total:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QListWidget" name="LivraisonPrevue"/>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="LivraisonPrevueLabel">
     <property name="text">
      <string>LivraisonPrevue   Total:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QListWidget" name="PCListe"/>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="PC">
     <property name="text">
      <string>PC(pour consulter Catalogue)</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QPushButton" name="ConsulterCatalogue">
     <property name="text">
      <string>Consulter Catalogue</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="ID">
     <property name="text">
      <string>ID:100</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QPushButton" name="EspacePersionnel">
     <property name="text">
      <string>Espace Personnel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
